src/mplhep/plot.py

```diff
@@ -4,6 +4,7 @@
 from collections import OrderedDict, namedtuple
 from typing import TYPE_CHECKING, Any, Union
 
+import hist as Hist
```

Comment:
is there a reason this needs to be renamed?

Reply:
`hist` is used [here](https://github.com/scikit-hep/mplhep/blob/199430685ff73788a942ab9862fd9590a258f297/src/mplhep/plot.py#L618)

Reply:
It would be better to rename that instead